client/components/DisplayTest/index.jsx

```diff
@@ -367,8 +355,20 @@ class DisplayTest extends Component {
             </nav>
         );
 
-        if (this.testHasResult) {
+        if (this.testResultsCompleted) {
             testContent = <TestResult testResult={test.results[testerId]} />;
+        } else if (this.testResultsSkipped) {
+            testContent = (
+                <TestIframe
+                    git_hash={git_hash}
+                    file={test.file}
+                    at_key={at_key}
+                    onResults={this.handleResults}
+                    ref={this.iframe}
+                    serializedForm={test.results[testerId].serialized_form}
+                    onSave={this.onSave}
```

Comment:
I think you no long have this function, right?

Reply:
oh right! removing